### PR TITLE
Remove duplicate regex in system.go for -check-requirements

### PR DESCRIPTION
**Pull request by redanaheim**
This fixes a minor issue where the same line is displayed twice in `opensnitchd -check-requirements`:

```
Checking system requirements for kernel version 6.8.9-asahi
------------------------------------------------------------------------------

        Checking => CONFIG_KPROBES=y
        Checking => CONFIG_KPROBES_ON_FTRACE=y
         - KPROBES not fully supported by this kernel.
        Checking => CONFIG_KPROBES_ON_FTRACE=y
         - KPROBES not fully supported by this kernel.
```

### CODE REVIEW

File: daemon/rule/operator.go
Comment (gustavo-iniguez-goya) [inline daemon/rule/operator.go:1]: Hi @redanaheim , this change is not related to this PR. Create a new PR for it if you think it improves rules parsing.

Comment (redanaheim) [reply daemon/rule/operator.go:1]: You're right, I just happened to commit those on my fork and forgot it would affect this PR.

Comment (redanaheim) [reply daemon/rule/operator.go:1]: @gustavo-iniguez-goya fixed.

**Comment by gustavo-iniguez-goya** [comment]
thank you @redanaheim !
